doc/topics/releases/fluorine.rst

```diff
@@ -870,3 +870,13 @@ for viewing minions, runners, and jobs as well as running execution modules
 and runners of a running Salt system through a REST API that returns JSON.
 See Salt-API_ documentation.
 .. _Salt-API: https://docs.saltstack.com/en/latest/topics/netapi/index.html
+
+Security
+========
+
+Windows runas changes
+---------------------
+
+A password is no longer required with ``runas`` under normal circumstances.
+The password option is only needed if the minion process is run under a
+restricted (non-administrator) account. 
```

Comment:
As I recall, password is only needed if it's under a non-admin account AND it's requesting priv escalation, right? If so, I think that should be called out specifically.

Reply:
@cachedout If the minion happens to be running under a non admin. They will need to use a password anytime the use the `runas` argument to `cmd.run`. Maybe this is better?

```
 A password is no longer required with ``runas`` under normal circumstances.
The password option is only needed if the minion process is run under a
restricted (non-administrator) account. In the aforementioned case, a password
is only required when using the ``runas`` argument to run command as a different 
user.
```